artemis-chart/templates/nginx-deployment.yaml

```diff
@@ -1,20 +1,22 @@
 {{- if not .Values.ingress.enabled -}}
-apiVersion: extensions/v1beta1
+apiVersion: {{ template "artemis.deployment.apiVersion" . }}
 kind: Deployment
 metadata:
   labels:
-    io.kompose.service: nginx
-    app: {{ .Release.Name }}
+    app: {{ .Release.Name }}-nginx
   name: {{ .Release.Name }}-nginx
 spec:
+  selector:
+    matchLabels:
+      app: {{ .Release.Name }}-nginx
   replicas: {{ .Values.replicaCount | default 1 }}
   strategy:
     type: Recreate
   template:
     metadata:
       creationTimestamp: null
       labels:
-        io.kompose.service: nginx
+        app: {{ .Release.Name }}-nginx
```

Comment:
shouldn't also the changes regarding
```
        {{- range $key, $value := .Values.nodeSelector }}
        {{ $key }}: {{ $value }}
        {{- end }}
      {{ toYaml . | nindent 8 }}
      {{- end }}
```
applied here as in the other deployments?